ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/draggable-seed.component.js

```diff
@@ -58,6 +58,7 @@ class DraggableSeed extends Component {
           'confirm-seed-phrase__seed-word--drop-hover': isOver && canDrop,
         })}
         onClick={onClick}
+        data-testid={`draggable-seed-${selected ? 'selected-' : ''}${word}`}
```

Comment:
Why do we need the `selected-` prefix on the test selector?

Reply:
It's to prevent it from being matched if it has already been selected (that's what the `and not(contains(@class, 'confirm-seed-phrase__seed-word--selected'))` part of the XPath selector was for).

Reply:
Oh, gotcha